corehq/apps/export/export.py

```diff
@@ -336,7 +336,8 @@ def get_export_documents(export_instance, filters):
     # We believe we can occasionally hit the 5m limit to process a single scroll window
     # with a window size of 1000 (https://manage.dimagi.com/default.asp?248384).
     # Thus, smaller window size is intentional
-    return query.size(500).scroll()
+    # Another option we have is to bump the "scroll" parameter up from "5m"
+    return query.size(200).scroll()
```

Comment:
Comment says that this is 1000

Reply:
I think the comment says that when it was 1000 we had problems, which is why it's now smaller.